### PR TITLE
[NIFI-2303] Update lineage styles

### PR DESCRIPTION
**Pull request by scottyaslan**

### CODE REVIEW

**Comment by joewitt** [comment]
reviewing

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css [hunk 1]
```diff
@@ -500,33 +524,40 @@ text.event-type.expand-parents, text.event-type.expand-children {
...
-    pointer-events: none;
-}
+/*path.flowfile-icon-mid-outline {*/
+    /*pointer-events: none;*/
+/*}*/
 
-path.flowfile-icon-top {
-    pointer-events: none;
-}
+/*path.flowfile-icon-arc {*/
```
Comment (joewitt) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css]: can this commented section be removed?

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css [hunk 2]
```diff
@@ -500,33 +524,40 @@ text.event-type.expand-parents, text.event-type.expand-children {
...
-    pointer-events: none;
-}
+/*path.flowfile-icon-mid {*/
+    /*pointer-events: none;*/
+/*}*/
 
-path.flowfile-icon-arc {
-    pointer-events: none;
-}
+/*path.flowfile-icon-mid-outline {*/
```
Comment (joewitt) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/provenance.css]: can this commented section be removed?

**Comment by joewitt** [comment]
Hovering over the icons in the lineage graph is extremely sensitive to have it highlight the path.  I think it is getting confused about when you're over the image in teh circle versus other parts of the circle.  When over image directly it doesn not highlight.  Could you take a look?

**Comment by joewitt** [comment]
Ok went through it all.  Can you check into the above comment on the icon blocking path highlighting.  Otherwise, looks good!

**Comment by joewitt** [comment]
Actually if you widen the screen the white background stops a sort of light gray background appears.
Also, there appears to be a way to close the screen in the top left of the lineage display window but unless you know it is there you cannot see it.  You can click on the whitespace area in the top right corner though and it nicely closes just that window and goes back to your search.

**Comment by joewitt** [comment]
also of note you can right click in the gray area when the screen widens and get the default context menu of the browser whereas if you stay over the white lineage area you cannot other than when right-clicking on an item (as expected and then it is a normal context menu)

**Comment by moranr** [comment]
I can confirm the issues @joewitt has mentioned. Additionally, the context menus still appear to be picking up some old styles. They should look just like the component context menus.

**Comment by moranr** [comment]
Also for the context menus I recommend changing the Expand icon to _fa-plus-square_ (from _fa-plus-square-o_) and Collapse to _fa-minus-square_ (from _fa-minus-square-o_); this reversed style will better match the others

**Comment by joewitt** [comment]
cool checking it out now

**Pull request closed without merging**